Miniscript-Template-000-Timelocks-in-Templates.mediawiki

```diff
@@ -0,0 +1,83 @@
+<pre>
+  Miniscript Template: 0
+  Title: Proposed Timelock Usage
+  Created: 2023-09-11
+</pre>
+
+
+1. Name of Template
+
+Proposed Timelock Usage
+
+2. Goal to be achieved by template
+
+[https://bitcoin.sipa.be/miniscript/ Miniscript] facilitates easier access to utilizing timelocks in Bitcoin, it opens a new frontier in Bitcoin security concerning the construction of more advanced Scripts. This document aims to standardize timelock application across different wallet solutions, focusing on wallet recovery and standardizing expected timelock usage for streamlined hardware and software wallet interfaces.
+
+In the event an output descriptor has been partially or fully lost, minimizing the overall search space for expected timestamp values can expedite recovery. General Miniscript usage supports any valid timelock value, this proposal seeks to guide implementation to more user-friendly practices.
+
+=Block height based Timelocks:=
+
+==Absolute block height based Timelocks==
+
+'''Proposal''': Set absolute block height-based timelocks as multiples of 100, always ending in 00.
+
+Examples of valid Block Height Absolute Timelocks
+
+* after(1000)
+* after(50700)
+* after(82800)
+* after(615000)
+
+
+== Relative block height based Timelocks ==
+
+'''Proposal''': Make relative block height-based timelocks multiples of 100, except for the maximum value,  65,536 (2^16).
+
+
+* older(100)
+* older(1500)
+* older(65535)
+
+
+=Epoch timestamp based Timelocks:=
+
+==Absolute Epochtime based Timelocks==
+
+'''Proposal''': To synchronize with real-world time rather than block time, employ epoch timestamps that are divisible by 43200 (Noon GMT) or 86400 (Midnight GMT). Optimally, use multiples of 604800 for Thursday at Midnight GMT.
+
+'''Limitation''': Avoid setting epoch timestamps beyond 2105 (4291704000) to prevent any possible issue with related to its 32 bit unsigned integer used for timestamps to happen in in February of 2106.
+
+Propsoed Examples of valid Epoch Timestamp Absolute Timelocks
```

Comment:
In e2167a422b8504dad625b538d3b3c39d55e39640:

Typo `Proposed`. But also I think you should just drop the word "Proposed" and make this a `===` subsubsection heading.

Reply:
Same elsewhere; change "Proposed Examples" to just "Examples".